modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -1277,17 +1271,6 @@ private MediaPackage checkForLegacyMediaPackageId(MediaPackage mp, Map<String, S
     }
   }
 
-  private void setPublicAclIfEmpty(MediaPackage mp) {
-    AccessControlList activeAcl = authorizationService.getActiveAcl(mp).getA();
-    if (activeAcl.getEntries().size() == 0) {
-      String anonymousRole = securityService.getOrganization().getAnonymousRole();
-      activeAcl = new AccessControlList(
-              new AccessControlEntry(anonymousRole, Permissions.Action.READ.toString(), true),
-              new AccessControlEntry(GLOBAL_CAPTURE_AGENT_ROLE, Permissions.Action.WRITE.toString(), true));
```

Comment:
@JulianKniephoff Have you tested this using an restricted access capture agent user? My guess is that processing would fail because of no access then.

Reply:
See, this is what I expected. x) No I haven't, I will try to put together a test later today or tomorrow. Of course if someone beats me to it, that would be appreciated. xP

Reply:
As the restricted access capture agent user is quite new: It will have the role GLOBAL_CAPTURE_AGENT ROLE and can be enabled / configured in https://github.com/opencast/opencast/blob/develop/etc/org.opencastproject.organization-mh_default_org.cfg#L71

Just do an ingest using that user and make sure a workflow is triggered (by ingest) that involves a WOH snapshot.

Reply:
(anyway: Absolutely +1 for removing the anonymous user's role here)

Reply:
The interesting thing, too, is that if you provide an ACL, the global capture agent role won't get access either, so things were probably already partially broken before, right? :thinking:

Reply:
Okay, so after an internal discussion with @lkiesow we decided to rebase this on `r/5.x` for now, so that it still gets into 5.1. This makes this discussion irrelevant for now (sorry @staubesv). @lkiesow promised to look into this issue further, though.